Source/WebKit/WebProcess/WebPage/WebPage.messages.in

```diff
@@ -506,6 +506,7 @@ GenerateSyntheticEditingCommand(enum:uint8_t WebKit::SyntheticEditingCommandType
 #if PLATFORM(MAC)
     AttributedSubstringForCharacterRangeAsync(struct WebKit::EditingRange range) -> (struct WebCore::AttributedString string, struct WebKit::EditingRange range)
     RequestAcceptsFirstMouse(int eventNumber, WebKit::WebMouseEvent event) AllowedWhenWaitingForSyncReplyDuringUnboundedIPC
+    SetCaretAppearance(int appearance)
```

Comment:
Would be nicer if this took a `CaretAnimatorType` parameter. Should be doable by adding something to `WebCoreArgumentCoders.serialization.in`.